tests/devices/i22/test_fswitch.py

```diff
@@ -26,3 +28,7 @@ async def test_reading_fswitch(fswitch: FSwitch):
             "value": 125,  # three filters out
         }
     }
+
+
+def test_fswitch_count_plan(RE: RunEngine, fswitch: FSwitch):
+    RE(count([fswitch]))
```

Comment:
Must: Check something about the value of the documens

Reply:
Here's a way it's been done in ophyd-async, I know there's a ticket either there, in dodal or blueapi to generalise
https://github.com/bluesky/ophyd-async/blob/main/tests/test_flyer_with_panda.py#L137